api/sip/client.py

```diff
@@ -682,10 +692,10 @@ def queue_response(self, response):
         self.responses.append(response)
 
     def connect(self):
-        # Since there is no socket, do nothing but reset the sequence
-        # number.
+        # Since there is no socket, do nothing but reset the local
+        # connection-specific variables.
         self.status.append("Creating new socket connection.")
-        self.sequence_number = 1
```

Comment:
The SIP2 spec doesn't say what number you're supposed to start the sequence number at, but I think 0 is a better choice than 1.